src/include/planner/create_plan.h

```diff
@@ -91,6 +121,13 @@ class CreatePlan : public AbstractPlan {
   // UNIQUE INDEX flag
   bool unique;
 
+  std::string trigger_name;
+  std::vector<std::string> trigger_funcname;
+  std::vector<std::string> trigger_args;
+  std::vector<std::string> trigger_columns;
+  expression::AbstractExpression* trigger_when = nullptr;
```

Comment:
It would be better to consider using a smart pointer (unique pointer in this case).